### PR TITLE
Restructure running dynamic checks

### PR DESCRIPTION
**Pull request by svetomir**
This PR unifies running dynamic checks across all checks.
It also implements new ping command needed for migration to this approach in order to preserve current approach, too.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,6 +13,7 @@ require (
 	github.com/spf13/viper v1.7.0
 	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
+	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f
```
Comment (janos) [inline go.mod:16]: I am not sure why is this dependency added. Probably by some auto import. Go mod tidy should help.

File: pkg/check/pingpong/ping.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	"fmt"
+
+	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/check"
+)
+
+// compile check whether Ping implements interface
+var _ check.Check = (*Ping)(nil)
+
+// Ping ...
```
Comment (janos) [inline pkg/check/pingpong/ping.go:14]: Update comments in this file.

**Review by janos — COMMENTED** [review]
Looks ok. Only minor comments.

File: cmd/beekeeper/cmd/check_ping.go [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+		nodeCount      int
+		image          string
+		persistence    bool
+		storageClass   string
+		storageRequest string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "ping",
+		Short: "",
```
Comment (acud) [inline cmd/beekeeper/cmd/check_ping.go:46]: missing descriptions

File: pkg/check/check.go [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+
+			rnd := random.PseudoGenerator(seed)
+			ng := cluster.NodeGroup(u.NodeGroup)
+			if err := updateNodeGroup(ctx, ng, u.Actions, rnd, i); err != nil {
+				return err
+			}
+		}
+
+		// wait at least 60s for deleted nodes to be removed from the peers list
+		time.Sleep(65 * time.Second)
```
Comment (acud) [inline pkg/check/check.go:63]: consider running this only if there was a removal of nodes

File: pkg/check/check.go [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+				return nil
+			})
+		}
+
+		if err := stageGroup.Wait(); err != nil {
+			return fmt.Errorf("stage %d failed: %w", i, err)
+		}
+
+		// wait 60s for deleted nodes to be removed from the peers list
+		time.Sleep(60 * time.Second)
```
Comment (acud) [inline pkg/check/check.go:113]: here too

**Review by acud — APPROVED** [review]
LGTM, minor comments

**Pull request merged**